dspace-server-webapp/src/test/java/org/dspace/app/rest/CommunityCollectionParentIT.java

```diff
@@ -0,0 +1,311 @@
+        map = mapper.readValue(content, Map.class);
+        String actualCommunityName = String.valueOf(map.get("name"));
+
+        assertThat(communityName, equalTo(actualCommunityName));
+        assertThat(communityUuidString, equalTo(String.valueOf(communityAA.getID())));
+        assertThat(communityUuidString, not(String.valueOf(communityA.getID())));
+        assertThat(communityUuidString, not(String.valueOf(communityAB.getID())));
+
+    }
+
+    @Test
+    public void col3ParentCommunityTest() throws Exception {
+
+        ObjectMapper mapper = new ObjectMapper();
+        String token = getAuthToken(admin.getEmail(), password);
+        MvcResult mvcResult = getClient(token)
+            .perform(get("/api/core/collections/" + col3.getID() + "/parentCommunity")).andReturn();
+
+        String content = mvcResult.getResponse().getContentAsString();
+        Map<String, Object> map = mapper.readValue(content, Map.class);
+        String communityUuidString = String.valueOf(map.get("uuid"));
+        String communityName = String.valueOf(map.get("name"));
+
+        mvcResult = getClient(token).perform(get("/api/core/communities/" + communityAB.getID())).andReturn();
+        content = mvcResult.getResponse().getContentAsString();
+        map = mapper.readValue(content, Map.class);
+        String actualCommunityName = String.valueOf(map.get("name"));
+
+        assertThat(communityName, equalTo(actualCommunityName));
+        assertThat(communityUuidString, equalTo(String.valueOf(communityAB.getID())));
+        assertThat(communityUuidString, not(String.valueOf(communityA.getID())));
+        assertThat(communityUuidString, not(String.valueOf(communityAA.getID())));
+
+    }
+
+    @Test
+    public void comAAParentCommunityTest() throws Exception {
+
+        ObjectMapper mapper = new ObjectMapper();
+        String token = getAuthToken(admin.getEmail(), password);
+        MvcResult mvcResult = getClient(token)
+            .perform(get("/api/core/communities/" + communityAA.getID() + "/parentCommunity")).andReturn();
+
+        String content = mvcResult.getResponse().getContentAsString();
+        Map<String, Object> map = mapper.readValue(content, Map.class);
+        String communityUuidString = String.valueOf(map.get("uuid"));
+        String communityName = String.valueOf(map.get("name"));
+
+        mvcResult = getClient(token).perform(get("/api/core/communities/" + communityA.getID())).andReturn();
+        content = mvcResult.getResponse().getContentAsString();
+        map = mapper.readValue(content, Map.class);
+        String actualCommunityName = String.valueOf(map.get("name"));
+
+        assertThat(communityName, equalTo(actualCommunityName));
+        assertThat(communityUuidString, equalTo(String.valueOf(communityA.getID())));
+        assertThat(communityUuidString, not(String.valueOf(communityB.getID())));
+
+    }
+
+
+    @Test
+    public void comABParentCommunityTest() throws Exception {
```

Comment:
I don't see any value in adding a more test for AB as this is the same test data category than AA